tests/src/test/scala/cats/tests/NonEmptyMapSuite.scala

```diff
@@ -209,4 +209,12 @@ class NonEmptyMapSuite extends CatsSuite {
       nem.toNel should ===(NonEmptyList.fromListUnsafe(nem.toSortedMap.toList))
     }
   }
+
+  test("NonEmptyMap#updateWith consistent with equivalent operations") {
+    forAll { (nem: NonEmptyMap[String, Int], i: (String, Int)) =>
+      nem.add(i) should ===(nem.add(i).updateWith(i._1)(identity))
```

Comment:
I think it would be better to have these three checks as separate tests :)

Reply:
Good point

Reply:
Separated the tests